File: Company_wise_Mock/Microsoft/Sort Colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int cnt[3] = {0, 0, 0};
        for(int e : nums){
            cnt[e]++;
        }
        
        int idx = 0;
        for(int j = 0; j < 3; j++)
        {
            for(int i = 0; i < cnt[j]; i++)
                nums[idx] = j, idx++;
        }
    }
};

File: Daily Challenges/2020/June/Sort Colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        cnt = [0] * 3
        
        for n in nums:
            cnt[n] += 1
        
        idx = 0
        
        for i in range(3):
            while cnt[i]:
                nums[idx] = i
                idx += 1
                cnt[i] -= 1
        
